apps/api-gateway/src/verification/verification.controller.ts

```diff
@@ -255,19 +248,12 @@ export class VerificationController {
     if (requestTypeV1 === ProofRequestType.PRESENTATIONEXCHANGE && !requestProof.presentationDefinition) {
             throw new BadRequestException(`type: ${requestTypeV1} requires presentationDefinition`);
     }
-        if (requestProof.proofFormats) {
-            const attributeArrayV1 = [];
-        for (const attrData of requestProof.proofFormats.indy.attributes) {
-          if (0 === attributeArrayV1.length) {
-            attributeArrayV1.push(Object.values(attrData)[0]);
-          } else if (!attributeArrayV1.includes(Object.values(attrData)[0])) {
-            attributeArrayV1.push(Object.values(attrData)[0]);
-          } else {
-            throw new BadRequestException('Please provide unique attribute names');
-          }           
 
-        }
-        }
+    
+      if (requestProof.proofFormats) {
```

Comment:
```suggestion
      if (ProofRequestType.INDY) {
```
Similar to above, this seems more appropriate